src/main/resources/db/changesets/20210301_revised_initial_schema.yaml

```diff
@@ -0,0 +1,246 @@
+# "Rebasing" the schema to a single changeset prior to production
+databaseChangeLog:
+- changeSet:
+    id: schema revision 2021
+    author: dd
+    changes:
+    - createTable:
+        tableName: workspace
+        remarks: |
+          There is one row in the workspace table for each workspace.
+        columns:
+          - column:
+              name: workspace_id
+              type: text
+              constraints:
+                primaryKey: true
+                nullable: false
+          - column:
+              name: display_name
+              type: text
+              remarks: |
+                A user-friendly name for the workspace. There is no guarantee of uniqueness across workspaces.
+                Can be null. Omitted for Rawls workspaces, since Rawls maintains its own name
+          - column:
+              name: description
+              type: text
+              remarks: |
+                Free-form text describing the workspace. Can be null. Omitted for Rawls workspaces, since Rawls
+                maintains its own name.
+          - column:
+              name: spend_profile
+              type: text
+              constraints:
+                nullable: true
+                unique: false
+          - column:
+              name: properties
+              type: jsonb
+              remarks: |
+                Key-value map to allow applications to store their data about the workspace
```

Comment:
what happens if two applications use the same key?

Reply:
I think we can come up with a key naming convention if the need arises.